torch/testing/_internal/common_methods_invocations.py

```diff
@@ -164,6 +164,7 @@ def __init__(self,
                  supports_sparse=False,  # supported for sparse
                  check_batched_grad=True,  # check batched grad when doing gradcheck
                  check_batched_gradgrad=True,  # check batched grad grad when doing gradgradcheck
+                 is_linear_map=False,  # supports in-place variant for Sparse Tensors
```

Comment:
nit: the comment is a little vague. Suggestion: `non-linear mappings on sparse tensors coalesce the inputs`

Reply:
Have made the changes as suggested